### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by Cayan** [RESTBench/requirements#2]
Should we add a requirement for any type of documentation generator tool?
Are we going to enforce an specific API documentation pattern (like Swagger)?

**Comment by igorsantos07** [comment]
This sounds like a hard requirement to be made, as some frameworks may provide this out of the box and some others don't - scaring those lighter guys out of the project.

This could be an optional requirement, such as HEAD/OPTIONS are.

**Comment by Cayan** [comment]
I agree this should be optional, this is an extra feature and would be most useful if a repository serves as an skeleton project.

The idea behind this issue is just to discuss an API documentation format and if it should be automated.
